DependencyInjection/Configuration.php

```diff
@@ -225,4 +227,25 @@ private function addContentListenerSection(NodeBuilder $nodeBuilder)
             ->end()
         ;
     }
+
+    /**
+     * Attach the form node to the tree.
+     *
+     * @param NodeBuilder $nodeBuilder
+     */
+    private function addFormSection($nodeBuilder)
+    {
+        $nodeBuilder
+           ->arrayNode('form')
+                ->canBeDisabled()
+                ->children()
+                    ->arrayNode('data_class')
```

Comment:
i think there is no need to fix xml config. that is done when you have a list of entries that in yml have a group name with an "s" at the end but in xml you can just have a list of elements with the name. as in: `sitemaps: {yml array}` vs `<sitemap>...</sitemap><sitemap></sitemap>`